test/integration_tests/conftest.py

```diff
@@ -40,6 +40,8 @@ def ctc_decoder():
     "fr": "20121212-0900-PLENARY-5-fr_20121212-11_37_04_10.flac",
     "it": "20170516-0900-PLENARY-16-it_20170516-18_56_31_1.flac",
 }
+_MIXTURE_FILENAME = "mix_3729-6852-0037_8463-287645-0000.wav"
+_EXPECTED_FILENAME = "expected_3729-6852-0037_8463-287645-0000.pt"
```

Comment:
This approach is problematic for couple of reasons;
1. The data is not easily recoverable.
2. The internal change of PyTorch can break this test anytime, and it is almost impossible to track down the exact reason, nor we want to spend time on it.

These are the reasons why TTS integration test does not have this kind of assertion.
I recommend to find alternative (not equivalent) way to gauge the correctness of the model.